docker-compose.yaml

```diff
@@ -65,3 +70,37 @@ services:
       retries: 5
       start_period: 20s
       timeout: 10s
+
+  #prometheus
+  prometheus:
+    image: prom/prometheus
+    ports:
+      - "9090:9090"
+    volumes:
+      - ./prometheus.yml/prometheus.yml:/etc/prometheus/prometheus.yml
```

Comment:
This volume mount looks confusing, make sure to rename the parent folder to something else.